packages/rum-core/src/performance-monitoring/transaction-service.js

```diff
@@ -147,7 +144,7 @@ class TransactionService {
 
     tr.captureTimings = true
 
-    if (type === PAGE_LOAD) {
+    if (tr.type === PAGE_LOAD) {
```

Comment:
There was a bug associated with how we redefine when we start managed transaction in frameworks and it did not have browserResponsive check set to false for page load transaction.

Reply:
We don't change the transaction type when we redefine so I'm not sure if I follow.

Reply:
when we call redefine the type would be undefined which would skip this if check which is the source of bug.